src/main/features/response/form/models/defendantPaymentPlan.ts

```diff
@@ -0,0 +1,86 @@
+import { LocalDate } from 'forms/models/localDate'
+import { PaymentSchedule } from 'ccj/form/models/paymentSchedule'
+import { IsDefined, IsIn, IsPositive, ValidateNested, MaxLength } from 'class-validator'
+import { IsValidYearFormat } from 'forms/validation/validators/isValidYearFormat'
+import { IsValidLocalDate } from 'forms/validation/validators/isValidLocalDate'
+import { IsFutureDate } from 'app/forms/validation/validators/dateFutureConstraint'
+import { IsLessThanOrEqualToSumOf } from 'forms/validation/validators/isLessThanOrEqualToSumOf'
+import { Fractions } from 'forms/validation/validators/fractions'
+import { ValidationErrors, RepaymentValidationErrors } from 'app/forms/validation/validationErrors'
+import { ValidationConstraints } from 'app/forms/validation/validationConstraints'
+
+export class DefendantPaymentPlan {
```

Comment:
There is ~90% overlap between this class and `RepaymentPlan`. To avoid duplicates subclass `DefendantPaymentPlan` from `RepaymentPlan`